Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -191,4 +196,31 @@ namespace AudioCommon
 		g_sound_stream->GetMixer()->StopLogDSPAudio();
 		s_audio_dump_start = false;
 	}
+
+	void IncreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume += offset;
+		if (currentVolume > AUDIO_VOLUME_MAX)
+			currentVolume = AUDIO_VOLUME_MAX;
+		UpdateSoundStream();
+	}
+
+	void DecreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
```

Comment:
Do we want to unmute on decrease volume?

Most TVs I see let you instantly mute something (say a sound which is way to loud) and then drop the volume to something more reasonable without it unmuting (and that loud sound resuming), only unmuting when you hit the mute button again, or increase the volume.